go/arrow/array/builder_test.go

```diff
@@ -19,9 +19,10 @@ package array
 import (
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+
 	"github.com/apache/arrow/go/v14/arrow/internal/testing/tools"
 	"github.com/apache/arrow/go/v14/arrow/memory"
-	"github.com/stretchr/testify/assert"
```

Comment:
just a slight annoyance here, I don't care as much about the order of the imports but can we remove the empty line?

Reply:
Oh, sorry. This is automatically `goimport`ed on my machine. Reverting.